ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,42 @@
+### Flow
+
+* [ ] Bug
+    * [ ] Can you reproduce the problem in [safe mode](https://atom.io/docs/latest/hacking-atom-debugging#check-if-the-problem-shows-up-in-safe-mode)?
+    * [ ] Did you check the [debugging guide](https://atom.io/docs/latest/hacking-atom-debugging)?
+    * [ ] Did you check the [FAQs on Discuss](https://discuss.atom.io/c/faq)?
+    * [ ] Are you reporting to the [correct repository](https://github.com/atom/atom/blob/master/CONTRIBUTING.md#atom-and-packages)?
+    * [ ] Did you [perform a cursory search](https://github.com/issues?q=is%3Aissue+user%3Aatom+-repo%3Aatom%2Felectron) to see if it is already reported?
+    * For more information on how to write a good bug report [look at the CONTRIBUTING guide](https://github.com/atom/atom/blob/master/CONTRIBUTING.md#how-do-i-submit-a-good-bug-report).
+* [ ] Enhancement
+    * [ ] Are you running the [latest version of Atom](https://atom.io/docs/latest/hacking-atom-debugging#update-to-the-latest-version)?
+    * [ ] Did you check to see if [a package](https://atom.io/packages) already offers the feature?
+    * [ ] Are you reporting to the [correct repository](https://github.com/atom/atom/blob/master/CONTRIBUTING.md#atom-and-packages)?
+    * [ ] Did you [perform a cursory search](https://github.com/issues?q=is%3Aissue+user%3Aatom+-repo%3Aatom%2Felectron) to see if it is already requested?
+    * For more recommendations on how to write a good enhancement suggestion [see the CONTRIBUTING guide](https://github.com/atom/atom/blob/master/CONTRIBUTING.md#how-do-i-submit-a-good-enhancement-suggestion).
```

Comment:
The idea for this is that it puts the common debugging steps that the triage team very often has to ask people about in follow up questions right into the issue up front. It is verbose ... but so is the back and forth on an Issue in a bunch of comments before we get at the real problem. I know that some people will skip by it ... or just check all the boxes without trying things, but I think many more will continue to not know the common things to help themselves if all we have is a single checkbox that points to a `CONTRIBUTING.md` that they already ignore the, admittedly, less prominent link to.